website/content/docs/platform/k8s/csi/index.mdx

```diff
@@ -0,0 +1,130 @@
+---
+layout: docs
+page_title: Vault CSI Provider
+sidebar_title: Vault CSI Provider
+description: >-
+  The Vault CSI Provider allows pods to consume Vault secrets using CSI Secret Store
+  volumes.
+---
+
+# Vault CSI Provider
+
+The Vault CSI Provider allows pods to consume Vault secrets using 
+[CSI Secret Store](https://github.com/kubernetes-sigs/secrets-store-csi-driver) volumes.
+
+~> The Vault CSI Provider requires the [CSI Secret Store](https://github.com/kubernetes-sigs/secrets-store-csi-driver) 
+  Driver to be installed.
+
+## Overview
+
+At a high level, the CSI Secret Store controller allows users to create `SecretStoreClass` objects. 
+This object defines which secret provider to use and what secrets to retrieve. When the provider 
```

Comment:
There is a sentence missing here. SPCs existing don't do anything on their own. The request is instigated by a pod mounting a CSI secrets store volume, which in turn specifies the SPC.

Reply:
Fixed 👍